README.rst

```diff
@@ -39,13 +39,20 @@ You can find our sources and single-click downloads:
 .. _Documentation: http://dipy.org
 .. _current trunk: http://github.com/nipy/dipy/archives/master
 
+
+Installing DIPY
+===============
+
+For installation instructions, please read our `documentation <http://nipy.org/dipy/installation.html>`_.
```

Comment:
I think we should start with
conda install -c conda-forge dipy vtk
and then provide the link to the rest of the installation information.